modules/cats/src/test/scala/pureconfig/module/cats/CatsSuite.scala

```diff
@@ -0,0 +1,23 @@
+package pureconfig.module.cats
+
+import cats.data.{ NonEmptyList, NonEmptyVector }
+import com.typesafe.config.ConfigFactory
+import org.scalatest._
+import pureconfig.syntax._
+
+class CatsSuite extends FlatSpec with Matchers with EitherValues {
+
+  case class Numbers(numbers: NonEmptyList[Int])
+
+  it should "be able to read a config with a NonEmptyList" in {
+    val config = ConfigFactory.parseString(s"""{ numbers: [1,2,3] }""")
+    config.to[Numbers].right.value shouldEqual Numbers(NonEmptyList(1, List(2, 3)))
```

Comment:
Just a minor nitpick: can you change this to `config.to[Numbers] shouldEqual Right(Numbers(NonEmptyList(1, List(2, 3))))`? That way, if an unexpected error occurs the test will complete with a pretty message from `shouldEqual` instead of throwing an exception.